src/libfuncs/enum.rs

```diff
@@ -359,33 +281,9 @@ pub fn build_match<'ctx, 'this>(
                         ))
                         .result(0)?
                         .into();
-                    let stack_ptr = helper
-                        .init_block()
-                        .append_operation(llvm::alloca(
-                            context,
-                            k1,
-                            llvm::r#type::opaque_pointer(context),
-                            location,
-                            AllocaOptions::new()
-                                .align(Some(IntegerAttribute::new(
-                                    IntegerType::new(context, 64).into(),
-                                    payload_layout.align() as i64,
-                                )))
-                                .elem_type(Some(TypeAttribute::new(payload_ty))),
-                        ))
-                        .result(0)?
-                        .into();
+                    let stack_ptr = helper.alloca(context, location, k1, Some(payload_layout.align()))?;
```

Comment:
should be `helper.init_block()` here.

Also you can use `alloca1` to avoid creating the k1 constant.

I also think you are missing some arguments there.

Reply:
fixed